docs/08_getting_started.md

```diff
@@ -0,0 +1,329 @@
+
+
+// HTTP client
+implementation "com.squareup.okhttp3:okhttp:3.12.13"
+```
+
+Note that you could use any http client other then okhttp, but it used here for more simplicity.
+
+
+### Make sure you use Java 11
+
+Change this in your module `build.gradle`:
+
+```gradle
+android {
+    ...
+    compileOptions {
+        // enabling desugaring seems required.
+        coreLibraryDesugaringEnabled true
+
+        // java version:
+        sourceCompatibility JavaVersion.VERSION_11
+        targetCompatibility JavaVersion.VERSION_11
+    }
+
+    // if you are using kotlin:
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_11
+        useIR = true
+    }
+}
+
+dependencies {
+    ...
+
+    // desugar
+    coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:1.1.5'
+}
+```
+
+
+### Proguard rules
+
+If you've `minifyEnabled` to `true` you should enable this rules:
+
+```pro
+# ....
+# For NewPipeExtractor
+-dontobfuscate
+-keep class org.schabi.newpipe.extractor.timeago.patterns.** { *; }
+-keep class org.ocpsoft.prettytime.i18n.** { *; }
+
+-keep class org.mozilla.javascript.** { *; }
+
+-keep class org.mozilla.classfile.ClassFileWriter
+-keep class com.google.android.exoplayer2.** { *; }
+
+-dontwarn org.mozilla.javascript.tools.**
+-dontwarn android.arch.util.paging.CountedDataSource
+-dontwarn android.arch.persistence.room.paging.LimitOffsetDataSource
+
+
+
+# Rules for OkHttp. Copy paste from https://github.com/square/okhttp
+-dontwarn okhttp3.**
+-dontwarn okio.**
+-dontwarn javax.annotation.**
+# A resource is loaded with a relative path so the package of this class must be preserved.
+-keepnames class okhttp3.internal.publicsuffix.PublicSuffixDatabase
+-keepclassmembers class * implements java.io.Serializable {
+    static final long serialVersionUID;
+    !static !transient <fields>;
+    private void writeObject(java.io.ObjectOutputStream);
+    private void readObject(java.io.ObjectInputStream);
+}
+```
+
+
+## Implement a Downloader
+
+The extractor needs to fetch websites and JSON files.
+Its entire network activity is going through the downloader you pass to it.  
+We have predefined some methods which need to be implemented.
+To get started, create a class extending
+[`Downloader`](https://github.com/TeamNewPipe/NewPipeExtractor/blob/master/extractor/src/main/java/org/schabi/newpipe/extractor/downloader/Downloader.java).
+We will call that class `DownloaderImpl` in this example.
+
+``` Java
+
+import org.schabi.newpipe.extractor.downloader.Downloader;
+
+public class DownloaderImpl extends Downloader {
+
+    public DownloaderImpl() {
+
+    }
+
+}
+
+```
+Your IDE will prompt you to implement at least one method:
+
+``` Java
+/**
+ * Do a request using the specified {@link Request} object.
+ *
+ * @return the result of the request
+ */
+public Response execute(@Nonnull Request request) throws IOException, ReCaptchaException {
+    // your code goes here
+}
+```
+
+<details>
+<summary>simple implementation</summary>
```

Comment:
referencing this thread: https://github.com/TeamNewPipe/documentation/pull/37/files#r784797034

The problem with copying it, is that it gets outdated unless active effort is put in to maintain it.
Since no extra commentary is added to the example, I would just remove it and just link it